### PR TITLE
Allow oe_num_mem_pages to be overridden

### PR DESCRIPTION
**Pull request by paulcallen**
Calculation of this setting is done via sighing and packaging.
If the unsigned version is used it uses the default enclave setting
which is not always big enough.
This change allows that setting to be updated for unsigned binaries.
This changed also updated algorithm a little as the reserved space was
not enough for a number of scenarios.

Signed-off-by: Paul Allen <paul.c.allen@microsoft.com>

### CODE REVIEW

File: tools/myst/host/utils.h [hunk 1]
```diff
@@ -34,4 +34,6 @@ int cli_getopt(
     const char* opt,
     const char** optarg);
 
+#define make_oe_num_heap_pages(rootfs_size) ((rootfs_size + (50 * 1024 * 1024)) / PAGE_SIZE)
```
Comment (mikbras) [inline tools/myst/host/utils.h:37]: Our coding conventions for macros is upper case.

**Review by mikbras — CHANGES_REQUESTED** [review]
Looks great. The only thing I could find is a nit. :) Our coding convention uses upper case for macros. You could make it an inline function for better type checking.

File: tools/myst/host/utils.h (hunk 1 above)
Comment (mikbras) [reply tools/myst/host/utils.h:37]:
```
MYST_INLINE size_t make_oe_num_heap_pages(size_t rootfs_size)
{
    const size_t extra_bytes = 50 * 1024 * 1024;
    return (rootfs_size + extra_bytes) / PAGE_SIZE;
}
```

**Comment by paulcallen** [comment]
Abandoning

**Pull request closed without merging**